cmd/controller/main.go

```diff
@@ -40,8 +41,12 @@ const (
 	defaultMultusGlobalNamespaces       = ""
 	defaultMultusKubeconfigPath         = "/etc/cni/net.d/multus.d/multus.kubeconfig"
 	defaultMultusLogFile                = ""
+	defaultMultusLogMaxSize             = 100 // megabytes
+	defaultMultusLogMaxAge              = 5   // days
+	defaultMultusLogMaxBackups          = 5
+	defaultMultusLogCompress            = true
 	defaultMultusLogLevel               = ""
-	defaultMultusLogToStdErr            = false
+	defaultMultusLogToStdErr            = true
```

Comment:
Could you revert it because of 'false' as default value?

Reply:
Sure!

Reply:
My previous thought about this is wrong.. :)